CIP-1694/README.md

```diff
@@ -420,22 +432,128 @@ However, this choice also limits the implementation effort for wallet providers
 
 In contrast to other protocol parameter updates, hard forks (or, currently, changes to the protocol's major version) require much more attention.
 Indeed, while other protocol parameter changes can be performed without significant software changes,
-a hard fork assumes that a supermajority of the network has upgraded the node to support the new set of features that are introduced by the upgrade. This means that the timing of a hard fork event must be communicated well ahead of time to all Cardano users, and requires coordination between stake pool operators, wallet providers, DApp developers, and the node release team.
+a hard fork assumes that a super majority of the network has upgraded the node to support the new set of features that are introduced by the upgrade. This means that the timing of a hard fork event must be communicated well ahead of time to all Cardano users, and requires coordination between stake pool operators, wallet providers, DApp developers, and the node release team.
 
 Hence, this proposal promotes hard fork initiations as a standalone governance action, separate from protocol parameter updates, as in the Shelley scheme.
 
 ### Treasury Withdrawals vs Project Catalyst
 
-Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one trasury withdrawal per epoch.
+Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one treasury withdrawal per epoch.
 
 Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
 
+### The purpose of the DReps
+
+Nothing in this proposal limits the SPOs from becoming DReps.
+Why do we have DReps at all?
+SPOs are chosen purely for block production.
+Voters can choose to delegate their vote to DReps without needing to consider whether they are
+also a good block producer.
+
+### AVST (action voting stake threshold)
+
+The AVST exists to ensure that the votes of the DReps are meaningful.
+For example, if only 10 Lovelace were properly delegated to DReps, and 9 Lovelace "vote yes" to
+a governance action, we would doubt the legitimacy of the result since there are
+billions of ADA in circulation.
+A high enough AVST legitimizes the representation.
```

Comment:
You know, now I'm confused. At the moment we say:
- The stake delegated to yes votes must be over a given fraction of the active voting stake
- The active voting stake must be over a given fraction of the total statke

We could just multiply those out, and say that "the stake delegated to yes votes must be over a given fraction of the _total_ state".

A guess the reason not to do this is that this way allows us to not always require the AVST be met. So if the DRep vote passes but doesn't meet the AVST, we can go to an SPO vote or fail. But we could also implement *that* more directly: just say that if the DRep vote fails, it can still be ratified if it passes an SPO vote (or not).

So it seems to me that we don't need AVST at all :thinking:

Reply:
> if the DRep vote fails, it can still be ratified if it passes an SPO vote (or not).

I think this is significantly different from the current semantics. For each governance action I believe AVST is a flag which indicates which group is actually in control of pass/veto decision making, in the early stages AVST will be low so SPOs will have control of governance actions, but as AVST rises DReps will be in control over governance actions. Which may not be preferable, shouldn't SPOs always have a say in hard forks and protocol parameter updates since it affects them the most.

In the above case DReps still have ultimate control since they can re-delegate to pools that also represent their voting preferences, but SPOs could act as a short term gate-keep to any bad decisions being made.

Reply:
As written, the SPOs can only vote if the AVST is not met. In particular, if "a legitimate DReps vote does not say yes", the SPOs do not get to vote. In your suggestion, however, the SPOs can ratify an action, even if "the will of the people does not say yes".